otlp/trace.go

```diff
@@ -347,6 +347,12 @@ func getSampleRate(attrs map[string]interface{}) int32 {
 			sampleRate = math.MaxInt32
 		}
 	}
+	// To make sampleRate consistent between Otel and Honeycomb, we coerce all 0 values to 1 here
+	// A value of 1 means the span was not sampled
+	// For full explanation, see https://app.asana.com/0/365940753298424/1201973146987622/f
+	if sampleRate == 0 {
+		sampleRate = 1
```

Comment:
This seems right... I notice we have a `defaultSampleRate` variable already declared as`int32(1)`. Any reason to not use that instead of hardcoding 1 here?

Reply:
I didn't notice that, thank you! I'll update to use that value 👍